src/sentry/dynamic_sampling/rules/utils.py

```diff
@@ -144,3 +150,16 @@ def get_enabled_user_biases(user_set_biases: Optional[List[ActivatableBias]]) ->
 
 def get_supported_biases_ids() -> Set[str]:
     return {bias["id"] for bias in DEFAULT_BIASES}
+
+
+def eval_dynamic_factor(base_sample_rate: float, x: float) -> float:
+    """
+    Function responsible for defining the sampling factor for a Relay rule.
+    The input range of the function must be within [0.0, 1.0] as it is designed to make the factor decay from x to 1.
+    """
+    if base_sample_rate < 0.0 or base_sample_rate > 1.0:
+        raise Exception(
+            "The dynamic factor function requires a sample rate in the interval [0.0, 1.0]."
+        )
+
+    return float(x / (x**base_sample_rate))
```

Comment:
I would put the whole dynamic factor logic in a separate PR. Let's first make sure that the switch from `rules` to `rulesV2` works before we make any functional changes.

Reply:
I agree, will try to update the PR by removing this.

Reply:
Should we default to 5 as factor? Or just leave the sample rate?

Reply:
I would keep the functional behavior exactly as it is on master branch, just rewritten in the new format. So I guess that means leave the sample rate as it was computed before?

Reply:
Perfect, will do.